src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -258,6 +258,16 @@ void IslandWindow::Initialize()
 
     _rootGrid = winrt::Windows::UI::Xaml::Controls::Grid();
     _source.Content(_rootGrid);
+
+    // initialize the taskbar object
+    HRESULT hr = CoCreateInstance(CLSID_TaskbarList, NULL, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(&_taskbar));
+    if (SUCCEEDED(hr))
+    {
+        if (SUCCEEDED(_taskbar->HrInit()))

```

Comment:
Maybe merge with the check above? Make it `SUCCEEDED(hr) && SUCCEEDED(_taskbar->HrInit())`